src/components/Icon.js

```diff
@@ -15,8 +15,19 @@ import { colors, normalize } from '../config';
 export const Icon = ({ xml, width, height, name, size, focused, iconColor, style, iconStyle }) => {
   const color = iconColor || (focused ? colors.accent : colors.primary);
 
+  const hitSlop = (normalize(44) - size ?? width) / 2;
```

Comment:
i am unsure what happens if we have a negative hitslop. did you check that?

if needed you should ensure that it is non negative.

Reply:
maybe solved in f90ab7f07ff605af93121f7569894af606fd71e3
this is my test : 
<img width="931" alt="Screenshot 2021-07-08 at 13 05 58" src="https://user-images.githubusercontent.com/43641321/124913994-186eed00-dff0-11eb-9d46-416748315d83.png">
output: 
<img width="789" alt="Screenshot 2021-07-08 at 13 05 20" src="https://user-images.githubusercontent.com/43641321/124914045-2886cc80-dff0-11eb-88a7-3d96cdeb0379.png">

Only question is why our size or width would be a negative value ?

Reply:
now solved in f90ab7f ?

Reply:
:+1: